README.md

```diff
@@ -0,0 +1,96 @@
+# Aspose.Imaging Cloud Python SDK 
+[Aspose.Imaging Cloud](https://products.aspose.cloud/imaging/cloud) is a true REST API that enables you to perform a wide range of image processing operations including creation, manipulation and conversion in the cloud, with zero initial costs. Our Cloud SDKs are wrappers around REST API in various programming languages, allowing you to process images in language of your choice quickly and easily, gaining all benefits of strong types and IDE highlights. 
```

Comment:
@s-zubov Please, change the link to https://products.aspose.cloud/imaging - this is a new requirement from the Cloud team which I agree with. Thanks.

Reply:
Thank you, I've updated the link.